index.html

```diff
@@ -220,16 +220,6 @@ <h3>What is a Verifiable Credential?</h3>
 presentations can be rapidly transmitted, making them more convenient than
 their physical counterparts when establishing trust at a distance.
       </p>
-        <p>
-            The persistence of digital information, and the ease with which
-            disparate sources of digital data may be collected and correlated,
-            comprise a privacy concern that the use of verifiable and easily
-            machine-readable credentials threatens to exacerbate. A <a href="#privacy-considerations">Privacy Considerations</a>
-            section addresses these issues. Items of particular concern in
-            this specification may also be noted in the text. Examples of
-            how to use this data model using privacy-enhancing technologies
-            such as zero-knowledge proofs are also provided.
-        </p>
```

Comment:
@David-Chadwick, did you mean to remove this?

Reply:
Certainly not. If I did I truly apologise

Reply:
I have just checked the master copy that I pulled as the above text is not in my original version. So I did not edit it out.